src/realm/realmReducer.js

```diff
@@ -286,6 +285,9 @@ export default (
             if (data.email_address_visibility !== undefined) {
               result.emailAddressVisibility = data.email_address_visibility;
             }
+            if (data.push_notifications_enabled !== undefined) {
+              result.pushNotificationsEnabled = data.push_notifications_enabled;
```

Comment:
nit: let's put these in the same order as the properties appear in the REGISTER_COMPLETE case above — that helps make it easy to compare the two and spot any gaps or discrepancies

Reply:
Doing that review now:
* There's what at first glance looks like a gap that `state.webPublicStreamsEnabled` doesn't get updated. But the story is that that's a server setting and there's no event to update it.
* That's it! Otherwise everything lines up.

I'll push a quick commit on top of the PR branch that makes explicit what's happening there. Then it should probably go before the main commit here.